RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -99,8 +102,10 @@ public CodeExplorerItemViewModel SelectedItem
                 OnPropertyChanged("CanExecuteIndenterCommand");
                 OnPropertyChanged("CanExecuteRenameCommand");
                 OnPropertyChanged("CanExecuteFindAllReferencesCommand");
+                OnPropertyChanged("CanExecuteExportAllCommand");
                 OnPropertyChanged("PanelTitle");
                 OnPropertyChanged("Description");
```

Comment:
Not a showstopper, but FYI since we're compiling with C# 6 these could be strongly-typed, refactor-friendly:

    OnPropertyChanged(nameof(<property>))